examples/eady_turbulence.jl

```diff
@@ -99,29 +99,31 @@
 # to stabilize the Eady problem and can be idealized as modeling the effect of
 # turbulent mixing below the grid scale. For both tracers and velocities we use
 # a Laplacian vertical diffusivity ``κ_z ∂_z^2 c`` and a horizontal
-# hyperdiffusivity ``ϰ_h (∂_x^4 + ∂_y^4) c``. 
+# hyperdiffusivity ``ϰ_h (∂_x^4 + ∂_y^4) c``.
 #
 # ### Eady problem summary and parameters
 #
 # To summarize, the Eady problem parameters along with the values we use in this example are
 #
 # | Parameter name | Description | Value | Units |
-# |:--------------:|:-----------:|:-----:|:-----:| 
+# |:--------------:|:-----------:|:-----:|:-----:|
 # | ``f``          | Coriolis parameter | ``10^{-4}`` | ``\mathrm{s^{-1}}`` |
 # | ``N``          | Buoyancy frequency (square root of ``\partial_z B``) | ``10^{-3}`` | ``\mathrm{s^{-1}}`` |
 # | ``\alpha``     | Background vertical shear ``\partial_z U`` | ``10^{-3}`` | ``\mathrm{s^{-1}}`` |
 # | ``c^D``        | Bottom quadratic drag coefficient | ``10^{-4}`` | none |
 # | ``κ_z``        | Laplacian vertical diffusivity | ``10^{-2}`` | ``\mathrm{m^2 s^{-1}}`` |
 # | ``ϰ_h``        | Biharmonic horizontal diffusivity | ``10^{-2} \times \Delta x^4 / \mathrm{day}`` | ``\mathrm{m^4 s^{-1}}`` |
 #
-# We start off by importing `Oceananigans`, `Printf`, and some convenient utils
-# for specifying dimensional constants:
+# We start off by importing `Oceananigans`, `Printf`, and some convenient constants
+# for specifying dimensional units:
 
-using Oceananigans, Oceananigans.Utils, Printf
+using Printf
```

Comment:
Do we need to add `Printf` in the "Install dependencies" section?

Reply:
Thankfully no since `Printf` is part of the standard library: https://docs.julialang.org/en/v1/stdlib/Printf/

Reply:
`Printf` is part of `Base` I think so doesn't need to be installed? `using Printf` seems to work for me on a clean environment.